Source/LibationAvalonia/Program.cs

```diff
@@ -13,8 +14,25 @@ namespace LibationAvalonia
 {
 	static class Program
 	{
-		static void Main()
+		static void Main(string[] args)
 		{
+			if (Configuration.IsMacOs && args != null && args.Length != 0 && args[0] == "hangover")
+			{
+				//Launch the Hangover app within the sandbox
+				Assembly asm = Assembly.GetExecutingAssembly();
+				string path = Path.GetDirectoryName(asm.Location);
+				Process.Start("Hangover" + (Configuration.IsWindows ? ".exe" : ""));
+				return;
+			}
+			if (Configuration.IsMacOs && args != null && args.Length != 0 && args[0] == "cli")
+			{
+				//Open a new Terminal in the sandbox
+				Assembly asm2 = Assembly.GetExecutingAssembly();
+				string libationProgramFiles = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+				Process.Start("/System/Applications/Utilities/Terminal.app/Contents/MacOS/Terminal", $"\"{libationProgramFiles}\"");
+				return;
+			}
```

Comment:
We can do this because we're already executing inside the sandbox. Any process created in the sandbox executes in the same sandbox. Unfortunately, all sandbox files are read/execute, so no writing!